$NetBSD: patch-ab,v 1.2 2010/08/12 12:31:47 asau Exp $

Hardwire paths to installed files and Java binary.
Fix shell.

--- bin/sbaz-setup.orig	2010-07-13 19:48:47.000000000 +0400
+++ bin/sbaz-setup	2010-08-12 16:21:31.000000000 +0400
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 set -e
 
@@ -10,23 +10,7 @@
 esac
 
 
-SOURCE=$0;
-SCRIPT=`basename "$SOURCE"`;
-while [ -h "$SOURCE" ]; do
-    SCRIPT=`basename "$SOURCE"`;
-    LOOKUP=`ls -ld "$SOURCE"`;
-    TARGET=`expr "$LOOKUP" : '.*-> \(.*\)$'`;
-    if expr "${TARGET:-.}/" : '/.*/$' > /dev/null; then
-        SOURCE=${TARGET:-.};
-    else
-        SOURCE=`dirname "$SOURCE"`/${TARGET:-.};
-    fi;
-done;
-if [ "$SCALA_HOME" = "" ]
-then
-    SCALA_HOME=`dirname "$SOURCE"`/..;
-    SCALA_HOME=`cd "$SCALA_HOME"; pwd`;
-fi
+SCALA_HOME=@SCALA_HOME@
 
 SBAZ="$SCALA_HOME/bin/sbaz"
 
